tests/converter/dialogflow_sample_project/stream.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
```

Comment:
Is this part of the DF flow sample project?

Reply:
good point, deleted.